src/blog/stylo-another-kind-of-rich-text-editor.md

```diff
@@ -0,0 +1,164 @@
+       paragraph,
+       container,
+       focus: 'first'
+     })
+  }
+};
+```
+
+Stylo exposes interfaces and utilities to ease the development of custom plugins. The most important being maybe transformParagraph which parses the elements to the DOM.
+
+Finally, it is worth to note the function createParagraphs is async which means the process of creating new elements can totally be hijacked by your application. As for example, when integrated in DeckDeckGo, the function does not always directly create new elements. For some more complex use cases, when the function is triggered, it opens modals that require more user interactions such as selecting an image before effectively modifying the content.
+
+![](https://cdn-images-1.medium.com/max/1600/1*9GgkGbx_4D-hIwT8PxJivQ.gif)
+
+*****
+
+#### Framework Agnostic 😎
+
+Stylo is a framework agnostic library of reusable components developed with [StencilJS](https://stenciljs.com/). It works on all modern browsers.
+
+*****
+
+#### Lightweight 🪶
+
+30kb gzipped might in certain circumstances already sound like a lot, specially to me, but in terms of WYSIWYG editor it actually turns out be lightweight.
+
+From what I observed, it might even be the lightest. There is also probably a bit of space for improvements.
+
+*****
+
+#### Future Proof 🚀
+
+While a handful of actions in the alpha version still rely on execCommand to apply styles (e.g. bold, italic) — the core of the library does not.
+
+Stylo controls what happens on every mutation. It uses the MutationObserver API to keep track of the changes and stack them in a custom “undo-redo” implementation. It also forward the information about what changed in the DOM to your application with custom events.
+
+Plugins are applied using plain vanilla JavaScript and the Selection API is also used to detect and apply transformations.
+
+That’s why, regardless of what will happen to the deprecated API, Stylo is built for the future.
+
+*****
+
+#### Open Source ⭐️
+
+The components are licensed under MIT license. They can be used by anyone, anywhere where the web is used and this even in a closed source application.
+
+To contribute and follow the adventure, watch and star the [GitHub repo](https://github.com/papyrs/stylo).
+
+*****
+
+### Is It Production Ready?
+
+![](https://cdn-images-1.medium.com/max/1600/1*snbd7Pn0GcdqaEa9m48cIQ.gif)
+
+Stylo is currently an **alpha** version! ⚠️ The project is under active development and contributions on GitHub are most welcome.
+
+I did use it — not without noticing many bugs I need to fix — to write this article. It might probably not be ready tomorrow nor the day after but I am confident that it will be ready to start the beta phase of our project that will run on the Internet Computer.
+
+*****
+
+### Conclusion
+
+I am well aware there is still quite some work to make [Stylo](https://stylojs.com) bulletproof but it is quite exciting because challenging 😉.
```

Comment:
```suggestion
I am well aware there is still quite some work to make [Stylo](https://stylojs.com) bulletproof but it is quite exciting because it is also challenging 😉.
```